resources/qml/delegates/Reply.qml

```diff
@@ -41,6 +42,40 @@ Item {
         gesturePolicy: TapHandler.ReleaseWithinBounds
     }
 
+    Platform.Menu {
```

Comment:
I don't know how to do that (no idea about how QT/QML works). Feel free to push code wherever you want.

Reply:
You just put that menu into a different file (the MessageView.qml, where the other Menu should be). That is all. It should just work, since that is a parent.

Reply:
Moved to MessageView.qml